### PR TITLE
docs(readme): edits to readme, pillar image added

### PR DESCRIPTION
**Pull request by dimakis**
### Description
Some have been made to the introduction, components and architecture sections of the README.
3 Pillar - SSV also added.
Moved images to resources dir.

### Type of change

~- [ ] Bug fix (non-breaking change which fixes an issue)~
~- [ ] New feature (non-breaking change which adds functionality)~
~- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)~
~- [ ] Other (please specify)~
- [x] Documentation change

### Checklist

~- [ ] Documentation added for the feature~
~- [ ] CI and all relevant tests are passing~
~- [ ] Verified independently by reviewer~
- [x] Code Review completed

## Screenshots (if appropriate):

### CODE REVIEW

**Comment by jackdelahunt** [comment]
Everything in the doc looks great just something to note is that the image is not being rendered in github.
image

File: README.md [hunk 1]
```diff
@@ -30,13 +31,16 @@ of the CLI ecosystem:
 - Configuration
 - Commands and Flags standards.
 
-Charmil SDK will typically be introduced into two different CLI
+Charmil SDK will typically be introduced into two different CLI:
```
Comment (jackdelahunt) [inline README.md:34]:
```suggestion
Charmil SDK will typically be introduced into two different CLIs:
```

File: README.md [hunk 2]
```diff
@@ -30,13 +31,16 @@ of the CLI ecosystem:
...
-- Host CLI - CLI that is being used by end user that can embed one or more Plugins CLI
-- Plugins CLI - Separate CLI that can be also embedded into host CLI
+- Host CLI - CLI that is being used by end user that can embed one or more plugin CLIs
+- Plugin CLIs - Separate CLIs that can be embedded into the host CLI
 
 
 ## Components
+<p align="center">
+  <img width="400" src="https://github.com/aerogear/charmil/resources/charmil-base-ssv-3-pillar.png">
+  </p>
```
Comment (jackdelahunt) [inline README.md:43]:
```suggestion
</p>
```
Very nit picky

**Review by jackdelahunt — COMMENTED** [review]
Great description and images for people trying to understand charmil. The changes I suggested are super small but they are quick to fix.

File: README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 
 <p align="center">
-  <img width="400" src="https://github.com/aerogear/charmil/raw/main/logo.png">
+  <img width="400" src="https://github.com/aerogear/charmil/resources/logo.png">
```
Comment (wtrocki) [inline README.md:9]:
```suggestion
  <img width="400" src="https://github.com/aerogear/charmil/raw/main/logo.png">
```

**Review by wtrocki — APPROVED** [review]

**Pull request merged**